src/components/blog/post/body/_heading.scss

```diff
@@ -0,0 +1,54 @@
+h1 {
```

Comment:
???

Reply:
This is an old fix in the Subvisual blog. Funny story, we used to allow users to use any headers they wanted, which meant many blog posts used multiple `h1` tags.

For SEO reasons, in most cases a page should not have more than one `h1` tag. But, since we're dealing with Markdown here, there's nothing stopping everyone from putting their own H1 in there, although we already have it in the blog post header (Markdown only generates the body).

This ensures such _meliantes_ don't screw up their own blog posts by mistake.

Reply:
So if someone by mistake adds an `#` thing on the mardown, that gets hidden right?

Reply:
Precisely.

Reply:
Makes sense